verifier/src/main/kotlin/tech/figure/classification/asset/verifier/event/defaults/DefaultVerifyAssetEventHandler.kt

```diff
@@ -55,29 +60,17 @@ object DefaultVerifyAssetEventHandler : AssetClassificationEventHandler {
             )
             return
         }
-        val scopeAttribute = try {
-            acClient.queryAssetScopeAttributeByScopeAddress(scopeAddress = scopeAddress, assetType = assetType)
-        } catch (t: Throwable) {
-            eventChannel.send(
-                EventIgnoredMissingScopeAttribute(
-                    event = event,
-                    eventType = this.eventType,
-                    message = "$messagePrefix Intercepted verification did not point to a scope with a scope attribute",
-                    t = t,
+        val onboardingStatus = event.assetOnboardingStatus
+            ?.takeIf { it in expectedOnboardingStatuses }
+            ?: run {
+                eventChannel.send(
```

Comment:
so this would just always run in the case of running against anything < 3.2.0, but I guess that is alright (part of the limited functionality you mentioned)

Reply:
Yessir.  Safe to use, but not happily lol